src/views/client/dashboard/components/createPostAlternativeFirstForm/createPostAlternativeFirstForm.jsx

```diff
@@ -1,106 +1,131 @@
+  setImages,
+  images,
+  handleOnChange,
+  setOnAdd,
+  cloudinaryModal,
 }) => {
-    const [isDisable, setIsDisable] = useState(true);
-
-    useMemo(() => {
-        setIsDisable(!(data.title.length > 0 && data.info.length > 0 && data.img.length > 0));
-    }, [data]);
+  const [isDisable, setIsDisable] = useState(true);
 
-    const handleNext = () => {
-        if (data.title === "") return errorHandler("You must provide a title");
-        if (data.info === "") return errorHandler("You must provide a post info");
-        if (data.img.length === 0)
-            return errorHandler("You must provide at list one photo");
-        setDisplay(true);
-    };
-    const closeUploadSection = () => {
-        setOnAdd(false);
-        if (images && images.length > 0) deleteOnCancel(images);
-    };
+  useMemo(() => {
+    setIsDisable(
+      !(data.title.length > 0 && data.info.length > 0 && data.img.length > 0)
+    );
+  }, [data]);
 
-    return (
-        <React.Fragment>
-            <form className="h-80 capitalize overflow-y-scroll">
-                <div className="flex flex-col w-full">
+  const handleNext = () => {
+    if (data.title === "") return errorHandler("You must provide a title");
+    if (data.info === "") return errorHandler("You must provide a post info");
+    if (data.img.length === 0)
+      return errorHandler("You must provide at list one photo");
+    setDisplay(true);
+  };
 
-                    {/* post title */}
-                    <input
-                        type="text"
-                        name="title"
-                        onChange={handleOnChange}
-                        placeholder="Post title"
-                        value={data?.title}
-                    />
+  const closeUploadSection = () => {
+    setOnAdd(false);
+    if (images && images.length > 0) deleteOnCancel(images);
+  };
 
-                    {/* description */}
-                    <textarea
-                        className="p-2 w-full border border-gray-200 rounded-lg"
-                        rows={3}
-                        onChange={handleOnChange}
-                        placeholder="What do you want to share?"
-                        name="info"
-                        value={data?.info}
-                    ></textarea>
+  const handleDeleteImg = (imgId) => {
+    const updatedImageList = images.filter((img) => img.img_id !== imgId);
+    setImages(updatedImageList);
+    deleteSingleImage(imgId);
+  };
 
-                    {/* term options */}
-                    <select
-                        onChange={handleOnChange}
-                        value={data?.range}
-                        className="m-0 capitalize"
-                        name="range"
-                    >
-                        <option value="short-term">short-term</option>
-                        <option value="long-term">long-term</option>
-                    </select>
-                </div>
+  return (
+    <React.Fragment>
```

Comment:
replace <React.Fragment> with empty brackets <></>
we are using <React.Fragment key="some-key"> just when we want to pass a key.